examples/pinproctest/pinproctest.cpp

```diff
@@ -46,12 +46,8 @@ PRResult LoadConfiguration(YAML::Node& yamlDoc, const char *yamlFilePath)
             fprintf(stderr, "YAML file not found: %s\n", yamlFilePath);
             return kPRFailure;
         }
-        YAML::Parser parser(fin);
         
-        while(parser) 
-        {
-            parser.GetNextDocument(yamlDoc);
-        }
+        yamlDoc = YAML::Load(fin);
     }
```

Comment:
The changes in this file were related to updates in `yaml-cpp`, but will go away when I remove that feature from `pinproctest`.